podcast/the-changelog-311.md

```diff
@@ -10,7 +10,7 @@ What I think is happening today in the cloud is we're kind of reimagining that i
 
 The next layer that we all kind of agreed on was Kubernetes, and the problem Kubernetes was solving is "Alright, I don't have one container, I have many of them. How do I deploy them and control them and scale them and keep them running, and kind of solve all the lifecycle problems that exist when I'm running a bunch of containers?" We've all agreed on that as the kind of de facto technology to solve that problem.
 
-Service mesh I think of as the third layer of this new platform that we're building... And the problem service mesh is trying to solve is "How do I actually control the interaction between different applications that I'm writing?" If containers let me write an application, in real life most interesting things are solved by having lots of pieces of applications talk to each other... So how do I actually see and understand how those different components talk to each other and have some control over that? At its heart, that's what service mesh is trying to do, is give us control over the integration between apps.
```

Comment:
I believe all you've changed here is added an extra space in `integration between apps`. Was this a mistake that you didn't mean to commit? Or maybe you changed something else here that I'm not seeing?

Reply:
Actually, I have replaced the word 'integration' with 'interaction'.

Reply:
Ahh! How'd I miss that? 😆